code-challenges.js

```diff
@@ -12,27 +12,39 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+const divThree = (num) => {
+  return num % 3 == 0 ? `${num} is divisible by three` : `${num} is not divisible by three`
+}
 
-
-
+console.log(divThree(num1));
+console.log(divThree(num2));
+console.log(divThree(num3));
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitilized = (array) =>  {
+  //concats the capitilized first character with the rest of the string.
+  return array.map(x => x.charAt(0).toUpperCase() + x.substring(1))
+}
 
-
-
+console.log(capitilized(randomNouns))
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const ascendingNums = (array) => {
+  //gets rid of non-number data types
+  array = array.filter(x => typeof x === "number");
+  //sorts numbers in ascending order
+  return array.sort((a,b) => a - b);
```

Comment:
great! you can even chain this all together into one line if you really want to! :)